### DISCUSSION TITLE
Getting this error "AttributeError: 'Options' object has no attribute 'filter_nsfw'"

### DISCUSSION THREAD

**Discussion by prateekagwl** [TheLastBen/fast-stable-diffusion#1034] [Q&A]
<img width="1444" alt="Screenshot 2022-12-15 at 3 20 43 PM" src="https://user-images.githubusercontent.com/7215052/207827993-6bbbd4af-0b8d-4eda-b297-d198eb19fe9d.png">

When I test a trained model (tried both on SD 1.5 and SD 2), I'm getting this error. How can I fix this?

**Comment by TheLastBen** [comment]
https://github.com/TheLastBen/fast-stable-diffusion/search?q=filter_nsfw&type=issues
